### PR TITLE
add SG and PS post

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: _posts/2021-01-18-sg-ps-update.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+site: [freiburg]
+title: Major update to SearchGUI and PeptideShaker
+tags:
+- tools
+---
+
+After 3 years of work, [SearchGUI (SG)](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/search_gui/),
+a tool that performs protein identification using various search engines, and [PeptideShaker (PS)](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/peptide_shaker/)
+ for protein identification which uses SearchGUI results have been deeply updated and released in a new major versions 4.0.7 and 2.0.5 respectively.
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:10]:
```suggestion
 for protein identification (which uses SearchGUI results) have been deeply updated and released in new major versions: 4.0.7 and 2.0.5, respectively.
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Furthermore, two new utility tools have been added to make the overall workflow more flexible:
+
+* [Identification Parameters](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/ident_params/) for creating parameter files reusable by both SG and PS
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:13]:
```suggestion
* [Identification parameters](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/ident_params/) for creating parameter files reusable by both SG and PS.
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* [FastaCLI](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/fasta_cli/) which appends decoy sequences to a fasta file in a format readable by SG and PS
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:14]:
```suggestion
* [FastaCLI](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/fasta_cli/), which appends decoy sequences to a FASTA file in a format readable by SG and PS.
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+The number of changes and improvements made is very large, a few highlights:
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:16]:
```suggestion
The number of changes and improvements made is large, a few highlights:
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+### SearchGUI:
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:18]:
```suggestion
### SearchGUI
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+* Support for spectrum files in mzML format
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:20]:
```suggestion
* Support for spectrum files in mzML format.
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+* Search engines updated to last versions. Options updated accordingly
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:21]:
```suggestion
* Search engines updated to the most recent versions. Options updated accordingly.
```

Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:21]: I am not sure that 'Options updated accordingly' means

File: _posts/2021-01-18-sg-ps-update.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+* FASTA file is now out of the search parameters
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:22]:
```suggestion
* FASTA file is now out of the search parameters.
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+* Many more supported search engines parameters
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:23]:
```suggestion
* Many more supported search engines parameters.
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+* more [detailed release notes](http://compomics.github.io/projects/searchgui/wiki/ReleaseNotes)
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:24]:
```suggestion
* More [detailed release notes](http://compomics.github.io/projects/searchgui/wiki/ReleaseNotes).
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+
+### PeptideShaker
+
+* Support for spectrum files in mzML format
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:29]:
```suggestion
* Support for spectrum files in mzML format.
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+* Internal refactoring, new internal db and more efficient data management
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:30]:
```suggestion
* Internal refactoring, new internal database and more efficient data management.
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+* Reduced file sizes and memory load
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:31]:
```suggestion
* Reduced file sizes and memory load.
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+* more [detailed release notes](http://compomics.github.io/projects/peptide-shaker/wiki/ReleaseNotes )
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:32]:
```suggestion
* More [detailed release notes](http://compomics.github.io/projects/peptide-shaker/wiki/ReleaseNotes).
```

File: _posts/2021-01-18-sg-ps-update.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+Thanks Carlos Horro Marcos for your awesome work on the Galxy integration.
```
Comment (beatrizserrano) [inline _posts/2021-01-18-sg-ps-update.md:34]:
```suggestion
Thanks, Carlos Horro Marcos for your awesome work on the Galaxy integration.
```

File: _posts/2021-01-18-sg-ps-update.md (hunk 7 above)
Comment (bgruening) [reply _posts/2021-01-18-sg-ps-update.md:21]: Parameters of the search engine

**Pull request merged**